ProjectName/Sources/Data/NetworkAPI/NetworkAPI.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  NetworkAPI.swift
+//
+
+import Foundation
+import Alamofire
+import RxSwift
+
+final class NetworkAPI: NetworkAPIProtocol {
+
+    private let decoder: JSONDecoder
+
+    init(decoder: JSONDecoder = JSONAPIDecoder.default) {
```

Comment:
We didn't add `JSONMapper` lib into this template, so I think we just need to use `JSONDecoder`. Please define the default instance also.
```suggestion
    init(decoder: JSONDecoder = JSONDecoder.default) {
```